src/instructions/Instructions.test.jsx

```diff
@@ -147,4 +147,74 @@ describe('SequenceExamWrapper', () => {
 
     expect(getByTestId('pending-prerequisites')).toBeInTheDocument();
   });
+
+  it('Instructions for error status', () => {
```

Comment:
left a comment about this test in another PR, we should make sure to test the escalation/contact link as well. We can merge this for now and cover it in that test PR just to keep things moving.